frontend/ui/font.lua

```diff
@@ -135,6 +136,35 @@ if G_reader_settings and G_reader_settings:has("font_ui_fallbacks") then
     logger.dbg("updated Font.fallbacks:", Font.fallbacks)
 end
 
+-- Match bold font to fallback by name. We do not use FontInfo name match
```

Comment:
May be add a first sentence to make it clear why all that:
-- We don't ship a bold variant (for most of?) our fallback fonts. Allow users to themselves drop a Noto Sans Bold variant of their most used fallbacks, and we will use them if present.

Reply:
Done.

Reply:
My `(for most of?)` was for you to possibly check if really no fallback font gets a bold font shipped - not to be included in the comment :)

Reply:
Fixed